hips/tiles/tile.py

```diff
@@ -95,7 +95,7 @@ def read(cls, meta: HipsTileMeta, filename: str = None) -> 'HipsTile':
 
         if meta.file_format == 'fits':
             hdu_list = fits.open(str(path))
-            data = hdu_list[0].data
+            data = hdu_list[0].data.astype('float')
```

Comment:
I just tried this and it prints `>i2`. So, I searched this up and it means 'big-endian byte integer', more [here](https://docs.scipy.org/doc/numpy/reference/generated/numpy.dtype.byteorder.html).